### PR TITLE
Describe 'AMQ_ANALYTICS_LOGGING_ENDPOINT' environment variable

### PR DESCRIPTION
**Pull request by pond**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
+
 gem 'github-pages'
-gem 'redcarpet'
\ No newline at end of file
+gem 'redcarpet'
```
Comment (pond) [inline Gemfile]: This was just some vertical white space tidying.

File: Gemfile.lock [hunk 2]
```diff
@@ -138,4 +138,4 @@ DEPENDENCIES
   redcarpet
 
 BUNDLED WITH
-   1.12.5
+   1.13.2
```
Comment (pond) [inline Gemfile.lock]: Routine maintenance bundle update.

File: Gemfile.lock [hunk 3]
```diff
@@ -21,14 +21,14 @@ GEM
     ffi (1.9.14)
```
Comment (pond) [inline Gemfile.lock]: Routine maintenance bundle update accounts for this file's changes.

**Comment by mbdietrich** [comment]
👍

**Pull request merged**